daemon/buildinfo.c

```diff
@@ -0,0 +1,216 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include <stdio.h>
+#include "./config.h"
+
+// Optional features
+
+#ifdef ENABLE_ACLK
+#define FEAT_CLOUD "YES"
+#else
+#define FEAT_CLOUD "NO"
+#endif
+
+#ifdef ENABLE_DBENGINE
+#define FEAT_DBENGINE "YES"
+#else
+#define FEAT_DBENGINE "NO"
+#endif
+
+#if HAVE_X509_VERIFY_PARAM_set1_host
+#define FEAT_TLS_HOST_VERIFY "YES"
+#else
+#define FEAT_TLS_HOST_VERIFY "NO"
+#endif
+
+#ifdef ENABLE_HTPS
```

Comment:
ENABLE_HTTPS ?

Reply:
It's set in `config.h` by the configure script if native HTTPS support is enabled.

Reply:
To clarify a bit further, all the `#ifdef` checks here are looking for things set in `config.h` by the configure script if certain features are enabled or specific libraries are going to be linked in. I tried a couple of different approaches when initially writing this, but this one looked the least ugly by a pretty significant margin.

Reply:
Stelios is probably talking about the spelling.

Reply:
Ah, yeah, missed that somehow...